src/tga_file.h

```diff
@@ -0,0 +1,60 @@
+
+
+#ifndef __TGA_FILE_H_INCLUDED__
+#define __TGA_FILE_H_INCLUDED__
+
+#include<iostream>
+#include<vector>
+#include<fstream>
+
+using namespace std;
+typedef char byte;
+
+class WriteTGA {
+public: 
+
+	byte header[18] = {
+		0,                  //1      id
+		0,                  //2      Color map type
+		0,                  //3      24bit uncompressed RGB
+		0, 0, 0, 0,		    //4-7    Color map related data
```

Comment:
Here should be 5 bytes responsible for color map not 4. See https://en.wikipedia.org/wiki/Truevision_TGA#Header.